src/config/config.go

```diff
@@ -30,6 +30,9 @@ var (
 	// ListTasks is a flag to print available tasks in a TaskFileLocation
 	ListTasks bool
 
+	// ListAllTasks is a flag to print available tasks in a TaskFileLocation
```

Comment:
Another opportunity for a refactor: neither `ListTasks` nor `ListAllTasks` are really configs, they're just flags. Maybe these can be removed from `config.go` and we just grab the flag directlyin `run.go`

Reply:
refactored in latest commit